index.html

```diff
@@ -5,11 +5,42 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <link rel="stylesheet" href="/teras-mavr-cv/assets/css/reset.css">
-
     <title>Document</title>
+
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/reset.css">
+    <link rel="stylesheet" href="../teras-mavr-cv/assets/css/style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300;400;500;700&display=swap" rel="stylesheet">
+
 </head>
 <body>
+  <!-- main_section -->
+  <div class="main_section">
+
+    <!--my sidebar-->
+    <aside class="my_sidebar">
+
+      <div class="my_img">
+        <img class="img" src="../teras-mavr-cv/self_photo.jpg" alt="photo">
```

Comment:
```suggestion
        <img class="img" src="assets/img/self_photo.jpg" alt="photo">
```
keep your files structured